features/admin/promotion/managing_coupons/filtering_coupons.feature

```diff
@@ -21,7 +21,7 @@ Feature: Filtering coupons
 
     @api @ui
     Scenario: Filtering coupons by code
-        Given I am browsing coupons of this promotion
-        When I filter by code containing "X"
+        When I am browsing coupons of this promotion
```

Comment:
Revert this change here and below, this step should be `Given`

Reply:
I proposed this change because `I am browsing coupons of this promotion` is actually part of the action taken within the system, not the test arrangement, and can instantly provide information about what type of step it is behind the scenes. However, I can understand that it might look odd, especially when we skip the Background reading